responsive.css

```diff
@@ -0,0 +1,210 @@
+
```

Comment:
- [ ] All over the project, most of the font stylings do not match.  Please, correct all the fonts that do not match styling.

## Explanation

In this module, you are required to follow the Figma template really closely.  Nothing is optional, and nothing is opinionated.  This includes all the fonts styling.

The reason for this is that you need to learn how to implement business requirements into software solutions.  This is the main goal.  Kindly treat the Figma template details as business requirements, not as optional suggestions.  If you follow this guidance, you will learn an ability that will be crucial for your career, because in a real-life job, is customary that the designers do the designing, and we do the software implementation according to their requirements.

## Hits and Further Advice

- You will need to check EACH text element on the Figma design, look at the font stylings, and compare if your styles match the ones on your project.  I myself did this for this review.  If they don't exactly match, please correct them.
- Font styles include font families, line-height, font-weight, font-size, letter-spacing, color.  Please, check each of them, nothing is optional.
- Ensure that the mobile view still works.
- I made 3 short videos for you, in order:

1. https://share.getcloudapp.com/xQuqLg10
2. https://share.getcloudapp.com/d5u9E0ZW
3. https://share.getcloudapp.com/rRu1768g

The last sentence I couldn't finish is:  "Also your project that way will look and feel like the Figma, and the software implementation will be perfect".

Please, use this help to your advantage and go ahead and correct your fonts! 💪 💪 💪